doc/introduction/connect.rst

```diff
@@ -0,0 +1,25 @@
+..
+    :copyright: Copyright (c) 2022 ftrack
+
+.. _introduction/connect:
+
+*******
```

Comment:
@henriknorin-ftrack having all was Parts could get confusing....
see whether you can wrap things around a common Part and use different hadings to make it easier to browse:

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections